chain/events/filter/event.go

```diff
@@ -28,7 +29,10 @@ func isIndexedValue(b uint8) bool {
 	return b&(types.EventFlagIndexedKey|types.EventFlagIndexedValue) > 0
 }
 
-type AddressResolver func(context.Context, abi.ActorID, *types.TipSet) (address.Address, bool)
+// AddressResolver is a function that resolves an actor ID to an address. If the
+// actor ID cannot be resolved to an address, the function should return
```

Comment:
```suggestion
// actor ID cannot be resolved to an address, the function must return
```
...because the code correctness depends on it?